src/components/RepoList.js

```diff
@@ -1,23 +1,25 @@
 import React, {PropTypes} from 'react';
 import RepoPreview from './RepoPreview';
 import {Route} from 'react-router-dom';
+import List, { ListItem, ListItemText } from 'material-ui/List';
+import Divider from 'material-ui/Divider';
```

Comment:
I need to take this out. I decided to not use dividers for every list item but did not remove the import.